packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2064,13 +2066,20 @@ function updateSuspensePrimaryChildren(
   if (currentFallbackChildFragment !== null) {
     // Delete the fallback child fragment
     currentFallbackChildFragment.nextEffect = null;
-    currentFallbackChildFragment.effectTag = Deletion;
+    currentFallbackChildFragment.effectTag =
+      (currentFallbackChildFragment.effectTag & StaticMask) | Deletion;
```

Comment:
Don't need the Deletion tag here because it goes on the parent  (`workInProgress` below).

Reply:
So I think this entire line can be removed.